x-pack/plugins/siem/public/resolver/view/index.tsx

```diff
@@ -21,9 +21,9 @@ import { ResolverEvent } from '../../../common/endpoint/types';
 
 const StyledPanel = styled(Panel)`
   position: absolute;
-  left: 1em;
-  top: 1em;
-  max-height: calc(100% - 2em);
+  left: 0;
+  top: 0;
+  bottom: 0;
```

Comment:
So it's a little closer to the latest round of comps, which have the table flush on the left side